app.json

```diff
@@ -0,0 +1,46 @@
+{
+  "expo": {
+    "plugins": [
+        [
+            "@config-plugins/react-native-ble-plx",
+            {
+                "isBackgroundEnabled":true,
+                "modes": ["peripheral", "central"],
+                "bluetoothAlwaysPermission": "Allow $(PRODUCT_NAME) to connect to bluetooth devices"
+            }    
+        ]
+    ],
+    "name": "bci-app",
+    "slug": "bci-app",
+    "version": "1.0.0",
+    "icon": "./assets/icon.png",
+    "userInterfaceStyle": "light",
+    "splash": {
+      "image": "./assets/splash.png",
+      "resizeMode": "contain",
+      "backgroundColor": "#ffffff"
+    },
+    "assetBundlePatterns": [
+      "**/*"
+    ],
+    "ios": {
+      "supportsTablet": true,
+      "bundleIdentifier": "com.anonymous.bci-app"
```

Comment:
Can we set this to `edu.miami.umnil.bci-app`?

Reply:
Pushed